lib/chat/ui/widgets/styles/bubble_message_styles.dart

```diff
@@ -2,55 +2,121 @@ import 'package:flutter/material.dart';
 import 'package:farmsmart_flutter/chat/ui/widgets/bubble_message.dart';
 
 class _Constants {
-  static const colorBlack = const Color(0xFF1B1B34);
-  static const colorWhiteAccented = const Color(0xFFF5F5FA);
-  static const colorGreen = const Color(0xFF00CD9F);
-  static const colorWhite = const Color(0xFFFFFFFF);
+  static const receivedTextColor = const Color(0xFF1A1B46);
+  static const receivedBackgroundColor = const Color(0xFFE9EAF2);
+  static const sentTextColor = const Color(0xFFFFFFFF);
```

Comment:
this seems like a theme. we could have a generic style builder that takes a theme.

so in different products you would just set the message bubble style base of the theme class.
myTheme = FarmsmartBubbleTheme().
 bubbleStyle = MessageBubbleStyle.fromTheme(myTheme);

Reply:
Should this will be changed on this PR or maybe we can leave it for refactor one @LeeMatthewHiggins ?
Also I haven't seen any Theme example in farm smart code base, is this something what has not been added yet?
Thanks!

Reply:
Maybe we can leave that for the refactor @LeeMatthewHiggins ?

Reply:
yep, let´s do it in the refactor.